### ISSUE TITLE
Cannot mint Kitty Items on emulator

### ISSUE THREAD

**Issue by alxflw** [onflow/kitty-items#212]
**Describe the bug**
Fresh KI installation, log in to service account and complete initialization. Go to the minting page (http://localhost:3001/admin/mint/). click mint.

The UI doesn't show an error but the "transaction sealed" status never changes. The console shows this error:
```
useMintAndList.js?0e5d:11 Uncaught (in promise) execution error code 1101: [Error Code: 1101] cadence runtime error Execution failed:
error: mismatched types
  --> 77320623f6e7242b82c227af9a1bc7a1f9f546a8d5cf001f104ad74fcefd973b:56:70
   |
56 |         let result = self.minter.mintNFT(recipient: receiver, typeID: typeID, rarityID: rarityID)
   |                                                                       ^^^^^^ expected `KittyItems.Kind`, got `UInt64`

error: mismatched types
  --> 77320623f6e7242b82c227af9a1bc7a1f9f546a8d5cf001f104ad74fcefd973b:56:88
   |
56 |         let result = self.minter.mintNFT(recipient: receiver, typeID: typeID, rarityID: rarityID)
   |                                                                                         ^^^^^^^^ expected `KittyItems.Rarity`, got `UInt64`

error: incorrect argument label
  --> 77320623f6e7242b82c227af9a1bc7a1f9f546a8d5cf001f104ad74fcefd973b:56:62
   |
56 |         let result = self.minter.mintNFT(recipient: receiver, typeID: typeID, rarityID: rarityID)
   |                                                               ^^^^^^ expected `kind`, got `typeID`

error: incorrect argument label
  --> 77320623f6e7242b82c227af9a1bc7a1f9f546a8d5cf001f104ad74fcefd973b:56:78
   |
56 |         let result = self.minter.mintNFT(recipient: receiver, typeID: typeID, rarityID: rarityID)
   |                                                                               ^^^^^^^^ expected `rarity`, got `rarityID`

error: cannot access `itemRarityPriceMap`: field has private access
  --> 77320623f6e7242b82c227af9a1bc7a1f9f546a8d5cf001f104ad74fcefd973b:61:20
   |
61 |             amount: KittyItems.itemRarityPriceMap[rarityID]!
   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: mismatched types
  --> 77320623f6e7242b82c227af9a1bc7a1f9f546a8d5cf001f104ad74fcefd973b:61:50
   |
61 |             amount: KittyItems.itemRarityPriceMap[rarityID]!
   |                                                   ^^^^^^^^ expected `KittyItems.Rarity`, got `UInt64`
```

**Expected behavior**
Minting works without any errors

**Comment by 10thfloor** [comment]
I was not able to reproduce this. I was able to initialize and mint without issues.
Is there anything else about the setup that might have caused this?

**Comment by alxflw** [comment]
weird. I thought I did a fresh install based on the latest of the repo. let me try again

**Comment by alxflw** [comment]
just completed it. still the same error. can someone else try to replicate? I don't know if my local setup is the reason

**Comment by bluesign** [comment]
Your api is out of date, clean docker images. 

Or you can try to start with: 'docker-compose --env-file .env.dev --profile dev up -d --build'

Let me know if it helps, if not I can try to help you debug more

**Comment by alxflw** [comment]
> Or you can try to start with: 'docker-compose --env-file .env.dev --profile dev up -d --build'

wow, that actually resolved the issue for me. thanks @bluesign! 

I wonder how common this might be? worth adding a troubleshooting section to the readme? if not, please close @10thfloor

**Comment by bluesign** [comment]
I think there is a related issue @alxflw , which will make kitty items more friendly.

https://github.com/onflow/kitty-items/issues/214

**Comment by 10thfloor** [comment]
Closing because resolved. Work is in progress to remove the dependency on Docker.
